### ISSUE TITLE
When benchmarking, keeps saying "Status: Failed" every 3 attempts

### ISSUE THREAD

**Issue by kyleboddy** [MultiPoolMiner/MultiPoolMiner#904]
Getting all sorts of errors when trying to benchmark... using the newest release. I am blocking castXMR because I am running six RX550s on this rig as a test which cast does not support. Basically for 3-4 benchmarks it keeps saying 0.00 PH/s, and some of the following errors:

NsGpuCNMiner.exe terminates with a zpool.ca CloudFlare error, bad gateway.
NsGpuCNMiner.exe terminates again due to "ErrorActionPreference" not existing, or not being able to find the path to a hashrate.txt file.
EthDcrMiner64.exe fails: "Exception calling ".ctor" with "2" argument(s): "No connection could be made because the target machine actively refused it 127.0.0.1:13333"" (ClaymoreAmdSia40)

It runs NsGpuCNMiner.exe again and says the ".ctor" no connection error string.

Basically keeps repeating those. The folder is on my desktop, I have both the VC++ files installed, Excavator in its place, etc. All the miners seem to have downloaded to the Bin folder just fine.

Any idea what the issues are? I checked the issues here and found nothing of note. This machine runs xmr-stak just fine 24/7 right now mining Cryptonight on Nicehash, so the architecture should work just fine.

**Comment by UselessGuru** [comment]
Logs, screenshots maybe?

**Comment by platinum4** [comment]
Is this still an ongoing issue?

**Comment by kyleboddy** [comment]
Unsure, moved on.
